acceptance-tests/src/main/java/io/blueocean/ath/api/classic/ClassicJobApi.java

```diff
@@ -89,28 +90,28 @@ public void deleteFolder(String folder) throws IOException {
 
     public void createFreeStyleJob(FolderJob folder, String jobName, String command) throws IOException {
         deletePipeline(folder, jobName);
-        URL url = Resources.getResource(this.getClass(), "freestyle.xml");
-        jenkins.createJob(folder, jobName, Resources.toString(url, Charsets.UTF_8).replace("{{command}}", command), true);
+        URL url = this.getClass().getResource("freestyle.xml");
```

Comment:
This makes for a different behavior from what it used to do.
`Resources.getResource` uses the `ClassLoader` of `Resources` i.e. the core class loader.
While `this.getClass().getResource` will use the plugin's classloader to search for the resource.

Maybe that difference doesn't matter here, but just so you are aware that there might be difference in behavior here and everywhere else.

A more "correct" translation of the old code might be:
```suggestion
        URL url = Jenkins.class.getResource("freestyle.xml");
```

Reply:
this is part of BlueOcean acceptance-tests module so it's not a plugin neither executed within Jenkins test framework. 
Definitely no need of this :)